packages/es-dev-server/src/command-line-args.ts

```diff
@@ -148,22 +148,20 @@ export const commandLineOptions = [
 
 /**
  * Reads command line args from arguments array. Defaults to `process.argv`.
- *
- * @param {string[]} [argv]
- * @param {{ defaultConfigDir?: string, defaultConfigName?: string }} config
- * @returns {import('./config.js').Config}
  */
-export function readCommandLineArgs(argv = process.argv, config = {}) {
+export function readCommandLineArgs(
+  argv = process.argv,
+  config: { defaultConfigDir?: string; defaultConfigName?: string } = {},
+) {
   const { defaultConfigDir = '.', defaultConfigName = 'es-dev-server.config.js' } = config;
   const dashesArgs = commandLineArgs(commandLineOptions, { argv, partial: true });
   const openInCommandLineArgs = 'open' in dashesArgs;
 
   // convert kebab-case to camelCase
-  /** @type {object} */
   const args = Object.keys(dashesArgs).reduce((acc, key) => {
     acc[camelcase(key)] = dashesArgs[key];
     return acc;
-  }, {});
+  }, {} as Record<string, string>);
```

Comment:
why not a concrete type?

Reply:
It's just a bag of properties, in the code that follows I go through the args and eventually output it in the format I need it to be.

Reply:
It's also how command-line-args returns it. I found the correct type for it (`commandLineArgs.CommandLineOptions`) so I'll use that.